BigCommerce/Integration/02_Particularities.md

```diff
@@ -0,0 +1,196 @@
+
+The import of offers is supported by the *BigCommerce* driver.
+- You can filter the orders that are to be imported. For detailed information, see [Configure BigCommerce connection](./01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).  
+- The *BigCommerce* connection enables parallel imports to handle larger loads.
+
+The driver manages the following data:
+- Contained products
+- Billing address
+- One shipping address      
+- Sales channels    
+  All sales channels that you connect via *BigCommerce* are handled as sub sales channels. This architecture enables you to run multiple stores in *BigCommerce*.
+
+Data that is not included in the above list is not included in the order import. For example, you may be interested in the following data, which is not part of the order import: 
+- Gift wraps
+- More than one shipping address
+- Customer IDs   
+   *Actindo* does not use the customer IDs of *BigCommerce* to prevent potential inconsistencies with already existing customer IDs. Instead, *Actindo* automatically creates a new customer ID, if the imported data does not match an existing *Actindo* customer.  
+
+If you have a strong need to have this data available, contact your *Actindo* MPS consultant.
+
+
+### Multiple Shipping Addresses
+
+Although *BigCommerce* allows multiple shipping addresses, *Actindo* supports the import of one shipping address only. Any orders that use multiple shipping addresses cannot be imported and will fail during the import.  
+For detailed information, see [Manage multiple shipping addresses](./03_ManageBigCommerceSettings.md#manage-multiple-shipping-addresses).
+
+### Discounts
+
+*BigCommerce* supports only one single discount coupon per order.  These coupons cannot be managed in the *Omni-Channel* module and subsequent processes in *Actindo*. For this reason, used discount coupons (name and code) are displayed in *Omni-Channel* as custom attributes on each line item.  
+*BigCommerce* processes discounts before taxes. To manage the discounts later in the *Accounting* module, the driver splits discounts on each line item, add them, and calculates a net discount value using the line items tax rate. Because of that even “discount in percent” discounts will be output as absolute values on each line item. 
+
+
+
+### Order status updates
+
+**Order delivery statuses**
+
+The driver supports updates on the order statuses, but *BigCommerce* can only manage a few *Omni-Channel* statuses. The order statuses are assigned as follows:
+
+| Status in Omni-Channel | Status in BigCommerce   
+   |-----|-------   
+   |Partially shipped|Partially shipped
+   |Ready for shipping|Awaiting shipment
+   |Shipped| Shipped   
+  
+
+**Order payment statuses**
+
+This feature is not supported, as *BigCommerce* does not allow an update of the payment status. This has no effects on subsequent processes in *Actindo* such as accounting. 
+
+
+
+## Shipments
```

Comment:
Gehört dieser Abschnitt unter ## Import of orders? Also, sollte es ### Shipments sein?

Reply:
Import of orders